packages/recommend-core/src/getBatchRecommendations.ts

```diff
@@ -0,0 +1,70 @@
+import { RecommendClient, RecommendationsQuery } from '@algolia/recommend';
+
+import { ProductRecord, RecordWithObjectID, TrendingFacet } from './types';
+import { mapByScoreToRecommendations, mapToRecommendations } from './utils';
+import { version } from './version';
+
+export type BatchKeyPair = {
+  key: string;
+  value: number;
+};
+
+export type GetBatchRecommendations<TObject> = {
+  keys: BatchKeyPair[];
+  queries: RecommendationsQuery[];
+  recommendClient: RecommendClient;
+  transformItems?: (
+    items: Array<ProductRecord<TObject>>
+  ) => Array<ProductRecord<TObject>>;
```

Comment:
We never apply `transformItems` in the batched recommendations?

Reply:
This is an oversight, I'll need to check with @marialungu about how `transformItems` is meant to be used.